package.json

```diff
@@ -5,6 +5,11 @@
   "source": "src/index.js",
   "main": "dist/cypress-nextjs-auth0.js",
   "types": "types/index.d.ts",
+  "files": [
+    "dist",
+    "types",
+    "encrypt.js"
```

Comment:
To be honest, I don't remember why `encrypt.js` is in a different file. You're welcome to explore whether this should be refactored. Consider this me sharing a thought and not making a request for this PR. :)

Reply:
I tried, but that lead to multiple issues in my demo repo (Cypress is not defined) and other errors. I will take a look at this in the future and leave it like this for now.